zokrates_cli/tests/code/sha_ethereum.code

```diff
@@ -0,0 +1,21 @@
+//This example creates the equivalent output as the following solidity code:
+//pragma solidity ^0.4.24;
+//contract SHA256Test {
+//        event Success(
+//        bytes32 indexed _id
+//    );
+//
+//    function calc_sha() public returns (bytes32)  {
+//        bytes32 a = 0x5;
+//        bytes32 b = 0x0;
+//        bytes32 result = sha256(b,a);
+//        emit Success(result);
+//        return result;
+//    }
+//}
+//
+//This code uses libsnark gadgets and needs to be compiled with the --gadgets flag enabled
+
+def main(field a) -> (field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field, field):
```

Comment:
As these were added to the tests (great), we need an `.arguments.json` file and an `.expected.witness` file to be able to run integration tests. Also `a` is not being used so could be removed.